openpype/hosts/traypublisher/plugins/create/create_editorial.py

```diff
@@ -239,35 +239,38 @@ def create(self, subset_name, instance_data, pre_create_data):
         sequence_path_data = pre_create_data["sequence_filepath_data"]
         media_path_data = pre_create_data["media_filepaths_data"]
 
-        sequence_path = self._get_path_from_file_data(sequence_path_data)
+        sequence_paths = self._get_path_from_file_data(
+            sequence_path_data, multi=True)
         media_path = self._get_path_from_file_data(media_path_data)
 
-        # get otio timeline
-        otio_timeline = self._create_otio_timeline(
-            sequence_path, fps)
+        for index, seq_path in enumerate(sequence_paths):
+            # get otio timeline
+            otio_timeline = self._create_otio_timeline(
+                seq_path, fps)
 
-        # Create all clip instances
-        clip_instance_properties.update({
-            "fps": fps,
-            "parent_asset_name": asset_name,
-            "variant": instance_data["variant"]
-        })
+            # Create all clip instances
+            clip_instance_properties.update({
+                "fps": fps,
+                "parent_asset_name": asset_name,
+                "variant": instance_data["variant"]
+            })
 
-        # create clip instances
-        self._get_clip_instances(
-            otio_timeline,
-            media_path,
-            clip_instance_properties,
-            family_presets=allowed_family_presets
+            # create clip instances
+            self._get_clip_instances(
+                otio_timeline,
+                media_path,
+                clip_instance_properties,
+                family_presets=allowed_family_presets
 
-        )
+            )
 
-        # create otio editorial instance
-        self._create_otio_instance(
-            subset_name, instance_data,
-            sequence_path, media_path,
-            otio_timeline
-        )
+            # create otio editorial instance
+            self._create_otio_instance(
+                subset_name + str(index),
```

Comment:
This is just opinion: I don't think this is the right way but can't think of different solution. Also I would keep previous logic -> If only 1 edl is passed the subset name does not change.

Reply:
This is resolved without any comment? This is change of logic whic his not backwards compatible, or?